python/its-vehicle/its_vehicle/client.py

```diff
@@ -193,7 +193,7 @@ def msg_cb(
                 payload_d.get("source_uuid", None) != self.cfg["instance-id"]
                 or self.cfg["mirror-self"]
             ):
-                self.mqtt_mirror.publish(topic, payload)
+                self.mqtt_mirror.publish(topic=topic, payload=payload)
```

Comment:
it really fix something?

Reply:
Yes, it does. As the commit log mentions, the API expects keyword-only arguments, not positional arguments.

And this fix *is* the one that fixes propagating messages to the local broker (issue #203).